library/src/webapp/js/sakai-iframe-copy-classes.js

```diff
@@ -1,4 +1,12 @@
 if (window.self !== window.top) {
-  document.body.classList = [...window.top.document.body.classList].join(' ')
-  document.getElementsByTagName("html")[0].classList = window.top.document.getElementsByTagName("html")[0].classList
+  var observer = new MutationObserver(function() {

```

Comment:
I'd make this a const. That way it will pass linting when we add it across the board.

Reply:
Changed to a const